src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -96,6 +94,14 @@
     @Column(name = "difficulty")
     private DifficultyLevel difficulty;
 
+    @Enumerated(EnumType.STRING)
+    @Column(name = "mode")
+    private ExerciseMode mode;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "team_scope")
+    private TeamScope teamScope;
```

Comment:
I guess here you can set the default value, so you will not have to set in the test helper classes. Also as far as I understand it can't be set to null.

Reply:
This is no longer relevant since the field has been removed (Stephan and I discussed that teams will always exist on an exercise scope for now. This might be extended back again at a later point).